build/scripts/build.ps1

```diff
@@ -180,22 +186,17 @@ function Run-MSBuild([string]$projectFilePath, [string]$buildArgs = "", [string]
 # Restore all of the projects that the repo consumes
 function Restore-Packages() {
     Write-Host "Restore using dotnet at $dotnet"
-
-    $all = @(
-        "Roslyn Toolset:build\ToolsetPackages\RoslynToolset.csproj",
-        "Roslyn:Roslyn.sln")
-
-    foreach ($cur in $all) {
-        $both = $cur.Split(':')
-        Write-Host "Restoring $($both[0])"
-        $projectFilePath = $both[1]
-        $projectFileName = [IO.Path]::GetFileNameWithoutExtension($projectFilePath)
-        $logFilePath = ""
-        if ($binaryLog) {
-            $logFilePath = Join-Path $logsDir "Restore-$($projectFileName).binlog"
-        }
-        Restore-Project $dotnet $both[1] $logFilePath
-    }
+    
+    Write-Host "Restoring Roslyn Toolset"
+    $logFilePath = if ($binaryLog) { Join-Path $logsDir "Restore-RoslynToolset.binlog" } else { "" }    
+    Restore-Project $dotnet "build\ToolsetPackages\RoslynToolset.csproj" $logFilePath
+
+    $logFilePath = if ($binaryLog) { Join-Path $logsDir "RepoToolsetBuild.binlog" } else { "" }        
```

Comment:
Please add ` Write-Host "Restoring Build.proj" for consistency with other restores.